source/_integrations/immich.markdown

```diff
@@ -36,6 +37,10 @@ Verify SSL certificate:
 
 The integration polls data every 60 seconds.
 
+## Media source
+
+A [media source](integrations/media_source) is provided for your user accessible [Immich](https://immich.app/) albums.
```

Comment:
what do you mean by "user accessible albums"?

Reply:
the whole immich integration is user scoped (_so you can have multiple config entries for the same immich instance, but different users_). This sentence should make clear, that the media source shows only that assets/albums, which are owned by or shared with the user (_those they are accessible for this particular used user_)

Reply:
any suggestions for improvements are highly welcome, as i'm not a native English speaker 🙈